src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -77,7 +77,7 @@ const PackageRecommendations = () => {
           },
         });
 
-        if (response && response.data && response.data.packages.length > 0) {
+        if (response && response.data && response.data.packages && response.data.packages.length > 0) {
```

Comment:
This is not related, but I was getting some errors, and I believe that we need to check for this too.